mysql.py

```diff
@@ -398,8 +398,8 @@ def fetch_mysql_slave_stats(conn):
 		if 'delay' in row and row['delay'] != None:
 			status['slave_lag'] = row['delay']
 
-	status['slave_running'] = 1 if slave_row['Slave_SQL_Running'] == 'Yes' else 0
-	status['slave_stopped'] = 1 if slave_row['Slave_SQL_Running'] != 'Yes' else 0
+	status['slave_running'] = 1 if slave_row['Slave_SQL_Running'] == 'Yes' and slave_row['Slave_IO_Running'] == 'Yes' else 0
+	status['slave_stopped'] = 1 if slave_row['Slave_SQL_Running'] != 'Yes' and slave_row['Slave_IO_Running'] != 'Yes' else 0
```

Comment:
I think this line should either be removed (we won't expose this metric anymore) or written with an OR:
status['slave_stopped'] = 1 if slave_row['Slave_SQL_Running'] != 'Yes' OR slave_row['Slave_IO_Running'] != 'Yes' else 0